curvefs/src/tools/README.md

```diff
@@ -0,0 +1,501 @@
+status: COPYSET_OP_STATUS_SUCCESS copysetStatus { state: 4 peer { address: "10.182.2.45:36701:0" } leader { address: "10.182.2.45:16701:0" } readonly: false term: 2 committedIndex: 1 knownAppliedIndex: 1 pendingIndex: 0 pendingQueueSize: 0 applyingIndex: 0 firstIndex: 1 lastIndex: 1 diskIndex: 1 epoch: 0 }
+
+...
+
+```
+
+[Back to Toc](#Contents)
+
+---
+
+## list
+
+### **list-fs**
+
+list all fs in cluster
+
+Usage:
+
+```shell
+curvefs_tool list-fs
+```
+
+Output:
+
+```shell
+fsId: 1
+fsName: "/test"
+status: INITED
+rootInodeId: 1
+capacity: 18446744073709551615
+blockSize: 1
+mountNum: 1
+mountpoints: "09a03e7f5ece:/usr/local/curvefs/client/mnt"
+fsType: TYPE_S3
+detail {
+  s3Info {
+    ak: "********************************"
+    sk: "********************************"
+    endpoint: "********************************"
+    bucketname: "********************************"
+    blockSize: 1048576
+    chunkSize: 4194304
+  }
+}
+
+...
+
+```
+
+### **list-coysetInfo**
+
+list all copysetInfo in cluster
+
+Usage:
+
+```shell
+curvefs_tool list-copysetInfo
+```
+
+Output:
+
+```shell
+copyset[4294967297]:
+statusCode: TOPO_OK
+copysetInfo {
+  poolId: 1
+  copysetId: 1
+  peers {
+    id: 1
+    address: "10.182.2.45:36701:0"
+  }
+  peers {
+    id: 2
+    address: "10.182.2.45:26701:0"
+  }
+  peers {
+    id: 3
+    address: "10.182.2.45:16701:0"
+  }
+  epoch: 0
+  leaderPeer {
+    id: 3
+    address: "10.182.2.45:16701:0"
+  }
+}
+
+...
+
+```
+
+[Back to Toc](#Contents)
+
+---
+
+## build
+
+### **build-topology**
+
+build cluster topology
+
+Usage:
+
+```shell
+curvefs_tool build-topology
+```
+
+Output:
+
+```shell
+
+```
+
+[Back to Toc](#Contents)
+
+---
+
+## umount
+
+### **umount-fs**
+
+umount fs from cluster
+
+Usage:
+
+```shell
+curvefs_tool umount-fs
+```
+
+Output:
+
+```shell
+umount fs from cluster success.
+```
+
+[Back to Toc](#Contents)
+
+---
+
+## usage
+
+### **usage-metadata**
+
+show the metadata usage of cluster
+
+Usage：
+
+```shell
+curvefs_tool delete-fs
+```
+
+Output:
+
+```shell
+metaserver[10.182.2.45:26701] usage: total: 1.06 TB used: 755.74 GB left: 327.23 GB
+metaserver[10.182.2.45:16701] usage: total: 1.06 TB used: 755.74 GB left: 327.23 GB
+metaserver[10.182.2.45:36701] usage: total: 1.06 TB used: 755.74 GB left: 327.23 GB
+all cluster usage: total: 3.17 TB used: 2.21 TB left: 981.68 GB
+```
+
+[Back to Toc](#Contents)
+
+---
+
+## delete
+
+### **delete-fs**
+
+delete fs by fsName
+
+Usage：
+
+```shell
+curvefs_tool delete-fs -fsName=/test -confirm
+```
+
+Output:
+
+```shell
+1. do you really want to delete fs (/test) :[Ny]y
+2. do you really want to delete fs (/test) :[Ny]y
+3. do you really want to delete fs (/test) :[Ny]y
+delete fs /test success.
+```
+
+[Back to Toc](#Contents)
+
+---
+
+## check
+
+### **check-copyset**
+
+checkout copyset status
+
+Usage:
+
+```shell
+curvefs_tool checkout-copyset -copysetId=10 -poolId=1
```

Comment:
Must specify copysetId and poolId ? Can check all copysets at a time?

Reply:
> 
poolId is necessary for request.
stautus-copyset can check all copysets at a time